src/Execution/Arguments/NestedOneToMany.php

```diff
@@ -67,4 +68,26 @@ public static function createUpdateUpsert(ArgumentSet $args, Relation $relation)
             }
         }
     }
+
+    /**
+     * @param  \Nuwave\Lighthouse\Execution\Arguments\ArgumentSet  $args
+     */
```

Comment:
```suggestion
```

This comment adds nothing but noise.

Reply:
Okie I will remove it, I just follow the `createUpdateUpsert` method, do you want me to remove it from both !?

Reply:
Done, I have modified it only from the new method I have added.